WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -357,7 +356,6 @@ public void onAttach(Activity activity) {
     /*
      * animate in the blog/tag info header after a brief delay
      */
-    @SuppressLint("NewApi")
```

Comment:
This annotation is still needed - `ViewAnimationUtils.createCircularReveal` (that's why travis is :crying_cat_face:)